app/src/main/java/org/wikipedia/history/HistoryFragment.java

```diff
@@ -578,31 +642,9 @@ protected Context getParentContext() {
         }
     }
 
-    private class MultiSelectCallback extends MultiSelectActionModeCallback {
-        @Override
-        public boolean onCreateActionMode(ActionMode mode, Menu menu) {
-            super.onCreateActionMode(mode, menu);
-            mode.getMenuInflater().inflate(R.menu.menu_action_mode_history, menu);
-            actionMode = mode;
-            selectedEntries.clear();
-            return super.onCreateActionMode(mode, menu);
-        }
-
-        @Override
-        protected void onDeleteSelected() {
-            deleteSelectedPages();
-            finishActionMode();
-        }
-
-        @Override
-        public void onDestroyActionMode(ActionMode mode) {
-            unselectAllPages();
-            actionMode = null;
-            super.onDestroyActionMode(mode);
-        }
-    }
-
     @Nullable private Callback callback() {
         return FragmentUtil.getCallback(this, Callback.class);
     }
+
+    private static class Search{ }
```

Comment:
Would it be better if name it as `SearchBar`? I was confused by the naming. Also, it would be good if adds a space after the name like: `SearchBar { }`